test/test_all/test.hpp

```diff
@@ -0,0 +1,27 @@
+//
+// test/test_all/test.cpp
+// PSim
+//
+// Contributors:
+//   Kyle Krol  kpk63@cornell.edu
+//
+// Pathfinder for Autonomous Navigation
+// Space Systems Design Studio
+// Cornell Univeristy
+//
+
+#ifndef PAN_PSIM_TEST_TEST_ALL_TEST_HPP_
+#define PAN_PSIM_TEST_TEST_ALL_TEST_HPP_
+
+#define UNITY_EXCLUDE_MATH_H
+#include <unity.h>
+#undef isnan
+#undef isinf
```

Comment:
If `unity.h` is not included in this fashion and you are compiling GNC source along with other unit tests, the compilation will likely fail due to the `isnan` macro! Just a heads up.